src/main/scala/com/azavea/modellab/Service.scala

```diff
@@ -54,7 +54,43 @@ object StaticConfig {
     .map(resourceToString)
     .map { str : String => str.parseJson }
 
-  val breaks = List("nlcd", "tr").zip(List("/breaks/nlcd.data", "/breaks/tr.data").map(resourceToString))
+  val breaks = List(
+    "black-0-1",
+    "black-0-100",
+    "blue-0-1",
+    "blue-0-100",
+    "cm",
+    "dosc",
+    "elevation",
+    "green-0-1",
+    "green-0-100",
+    "landsat",
+    "nlcd",
+    "nlcd-reclass",
+    "purple-0-100000",
+    "red-0-1000",
+    "rosc",
+    "tpi",
+    "tr"
+  ).zip(List(
+    "/breaks/black-0-1.data",
```

Comment:
It could a list of tuples to start with if you want to look a little prettier:
```scala
List(
   ("black-0-1", "/breaks/black-0-1.data"),
   ...
).map(resourceToString)